src/server/scripts/Spells/spell_rogue.cpp

```diff
@@ -880,10 +882,26 @@ class spell_rog_stealth : public SpellScriptLoader
                     target->CastSpell(target, SPELL_ROGUE_OVERKILL_PERIODIC, true);
             }
 
+            void HandleSpeedIncrease(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/) {
+
```

Comment:
same here. CalcValue. Whenever you use  SetSpeed, it's most likely a hack because these helpers are only meant to be used by internals.